cypher-shell/src/test/java/org/neo4j/shell/cli/CliArgHelperTest.java

```diff
@@ -179,4 +187,43 @@ public void allowsEncryptionToBeTurnedOnOrOff() {
         assertEquals(true, CliArgHelper.parse("--encryption", "true").getEncryption());
         assertEquals(false, CliArgHelper.parse("--encryption", "false").getEncryption());
     }
+
+    @Test
+    public void shouldParseSingleIntegerArgWithAddition() {
+        CliArgs cliArgs = CliArgHelper.parse( "-P", "foo=>3+5" );
```

Comment:
It would be reassuring to have a test that also show that it works to have whitespaces, e.g. `foo => 3 + 5`